src/form/bf_mx.jl

```diff
@@ -1,5 +1,5 @@
-import LinearAlgebra: diag
-
+import LinearAlgebra: diag, diagm
+import MathOptInterface
```

Comment:
MathOptInterface should be imported in `src/PowerModelsDistribution.jl`, the LinearAlgebra ones were done this way to make it clear where specific math functions were coming from in each file

Reply:
This import can be avoided if JuMP exports PowerCone like it does for SecondOrderCone and PSDCone.

@ccoffrin I vaguely remember you mentioned at some point to open an issue on JuMP.jl for this. What should I ask for? To export PowerCone in a way similar to PSDCone?

Reply:
For now I resolved it by moving the import as suggested by David.